kiva/agg/freetype2/src/tools/docmaker/sources.py

```diff
@@ -208,7 +208,7 @@ def  __init__( self, processor, filename, lineno, lines ):
         self.format    = processor.format
         self.content   = []
 
-        if self.format == None:
+        if self.format is None:
```

Comment:
This file is not relevant. It just happens to be in the vendored freetype tree that's here.

Reply:
Ah I see, should I revert this change then, or does it not matter

Reply:
Yeah, I would leave all the vendored code alone